lucille-core/src/main/java/com/kmwllc/lucille/stage/ParseJson.java

```diff
@@ -76,10 +75,23 @@ public Iterator<Document> processDocument(Document doc) throws StageException {
       ctx = this.jsonParseCtx.parse(doc.getString(this.src));
     }
     for (Entry<String, Object> entry : this.jsonFieldPaths.entrySet()) {
-      JsonNode val = ctx.read((String) entry.getValue(), JsonNode.class);
-      doc.setField(entry.getKey(), val);
+      if (isValidEntry(entry)) { // makes sure that key and value are not null
```

Comment:
suggest validating the json field paths in start(), logging a warning for invalid entries and removing invalid entries from the list